README.md

```diff
@@ -1,39 +1,22 @@
-# talon_one
```

Comment:
These changes must be traced back in the sdks generation folder in talon-service, otherwise - the next version generated will override all these beautiful changes.

We should add a README template (like we have [here](https://github.com/talon-one/talon-service/blob/4a1c2964abae623f94b0b3718ddc71633a09e091/sdks/bundle/codegen/php/templates/README.mustache#L1) for PHP and add it in the python sub-folder, next to [this](https://github.com/talon-one/talon-service/blob/4a1c2964abae623f94b0b3718ddc71633a09e091/sdks/bundle/codegen/python/config.json#L1)

The mustache template should be [this version](https://github.com/OpenAPITools/openapi-generator/blob/v4.3.1/modules/openapi-generator/src/main/resources/python/README.mustache) // Sharing because I know how painful finding this could be 😅